modular_azurepeak/code/modules/spells/pantheon/inhumen/zizo.dm

```diff
@@ -178,16 +178,212 @@
 	miracle = TRUE
 	devotion_cost = 30
 	range = 2
-	
+
 /obj/effect/proc_holder/spell/self/zizo_snuff/cast(list/targets, mob/user = usr)
 	. = ..()
 	if(!ishuman(user))
 		revert_cast()
 		return FALSE
 	var/checkrange = (range + user.get_skill_level(/datum/skill/magic/holy)) //+1 range per holy skill up to a potential of 8.
-	for(var/obj/O in range(checkrange, user))	
+	for(var/obj/O in range(checkrange, user))
 		O.extinguish()
 	for(var/mob/M in range(checkrange, user))
 		for(var/obj/O in M.contents)
 			O.extinguish()
 	return TRUE
+
+/obj/effect/proc_holder/spell/invoked/zizo_silence
+
+	name = "Silence of Progress"
+	desc = "An unholy hush that stifles prayer and mercy alike. The stillness clicks with Zizo's design."
+	overlay_state = "zizosilence"
+	clothes_req = FALSE
+	releasedrain = 30
+	chargedrain = 0
+	chargetime = 0
+	range = 7
+	warnie = "sydwarning"
+	movement_interrupt = FALSE
+	sound = 'sound/magic/churn.ogg'
+	invocation = "This is the sound of progress!"
+	invocation_type = "shout"
+	associated_skill = /datum/skill/magic/holy
+	devotion_cost = 20
+	recharge_time = 20 SECONDS
+	miracle = TRUE
+
+/obj/effect/proc_holder/spell/invoked/sacred_silence/cast(list/targets, mob/user = usr)
+	if(!isliving(targets[1]))
+		revert_cast()
+		return FALSE
+
+	var/mob/living/target = targets[1]
+	if(target.anti_magic_check(TRUE, TRUE))
+		return FALSE
+
+	target.visible_message(
+		span_warning("[user] sketches a crooked sigil in the air - the sound around [target] stutters and dies!" ),
+		span_warning("A cold, clockwork hush clamps my throat - prayers turn to static!")
+	)
+
+	var/skill = max(1, user.get_skill_level(associated_skill))
+	var/dur_s  = clamp(skill * 4, 4, 20)
+	var/dur_ds = dur_s SECONDS
+
+	if(istype(target, /mob/living/carbon/human))
+		var/mob/living/carbon/human/H = target
+		if(H.devotion && istype(H.devotion.patron, /datum/patron/divine))
+			dur_ds *= 2
+
+			var/tier = max(1, H.devotion.level)
+			var/drain = 50 * tier
```

Comment:
Why are higher tiered divine clerics being punished harsher by a t0 miracle? It should be the opposite. Start at a higher number and divide by the tier so that lower tiered clerics get owned.